### PR TITLE
Add input label before/after pseudo-elements css mixins

### LINKED ISSUE #702: Add variables to style label before/after pseudo-elements
### Description
Want to have possibility to style label after/before pseudo-elements. (ex. It's a great way to add required asterisk.)

### Expected outcome

Have possibility to style label after/before pseudo-elements.

### Actual outcome

There is no way to style those pseudo-elements.

### Browsers Affected

- [x] Chrome
- [x] Firefox
- [x] Safari 9
- [x] Safari 8
- [x] Safari 7
- [x] Edge
- [x] IE 11
- [x] IE 10

### PR DESCRIPTION
**Pull request by ydubinskyi**
Fixed #702

### CODE REVIEW

**Comment by ydubinskyi** [comment]
@googlebot I signed it!

**Comment by ydubinskyi** [comment]
@googlebot I fixed it.

File: paper-input.js [hunk 1]
```diff
@@ -158,6 +158,14 @@ Polymer({
       label {
         pointer-events: none;
       }
+
+      label:before {
```
Comment (e111077) [inline paper-input.js]: Can you please move this inside `paper-input-container`? perhaps `::slotted(*::before)` (i don't remember if `::before` goes in or outside the parens)

Comment (ydubinskyi) [reply paper-input.js]: Done

**Comment by e111077** [comment]
published as 3.2.0

**Pull request merged**